docker_images/grafana_nginx/htpasswd

```diff
@@ -0,0 +1,2 @@
+grafana:$apr1$u8e6RJMa$ThJKHrclQ2EZCz3gmUcKT/
```

Comment:
Hashed committed pwd

Reply:
let's leave it as is for now to finish with fully buildable code.
We have plan to move htpasswd file generation to the runtime (from overlay file) once we update deployment charts

Reply:
Filed https://github.com/hystax/optscale/issues/132